### ISSUE TITLE
[FR] Add support for zone or device_tracker entity 

### ISSUE THREAD

**Issue by Scoop2389** [jaydeethree/Home-Assistant-weatherdotcom#18]
Allow the option to provide a device_tracker or zone entity instead of lat long at setup. The latitude and longitude of the device/zone will be used instead of it being hardcoded into the integration settings, allowing for the location of the weather to change with a zone or device. Similar to the [nws_alerts integration](https://github.com/finity69x2/nws_alerts)

**Comment by jaydeethree** [comment]
Unfortunately I don't have enough spare time to work on this at the moment. If someone wants to create a PR that adds this feature then I'd be happy to review that, but right now my time is just too limited to work on this request. To be clear - I still intend to keep this integration working (I personally rely on it for a lot of my Home Assistant scripts/automations so it's important to me 😄), I'm just scaling back on feature requests.

**Comment by zigzag930** [comment]
I sure would like to see this feature also.  We travel in a RV and it would be nice if the weather location would update based on our Lat/Lon.

**Comment by jaydeethree** [comment]
Yeah, I definitely think there are some good use cases for this, but I haven't taken the time to implement it because it's not something that I would personally use. I might get to this someday, but I'm also still happy to review a PR if someone else wants to try implementing it.

**Comment by zigzag930** [comment]
Thank you for your reply.  If and when you do decide to implement - please let me know.  I’d be happy to help test or assist in any way.

I was a C programmer for many years, but haven’t coded in at least 10 years - so doubtful, I’d be able to takle this.

Anyway, thanks again for your reply.

Jim

**Comment by jaydeethree** [comment]
This has been implemented in https://github.com/jaydeethree/Home-Assistant-weatherdotcom/pull/71 and will be included in the next release (hopefully a few days from now).
